libpod/networking_linux.go

```diff
@@ -402,7 +402,7 @@ func (r *Runtime) GetRootlessNetNs(new bool) (*RootlessNetNS, error) {
 	// the cleanup will check if there are running containers
 	// if you run a several libpod instances with different root/runroot directories this check will fail
 	// we want one netns for each libpod static dir so we use the hash to prevent name collisions
-	hash := sha1.Sum([]byte(r.config.Engine.StaticDir))
+	hash := sha256.Sum256([]byte(r.config.Engine.StaticDir))
```

Comment:
@Luap99 Any chance this breaks existing containers after an upgrade?

Reply:
Yes this will likely break connectivity between old containers started before update and containers started after update.
It will also fail to cleanup the old netns. However a reboot will fix this so I think we can accept this for a 4.1 change.

Reply:
Ack, SGTM. Maybe just recommend folks restart rootless containers after the update? Cleanup will be broken, but aside from leaking a single network namespace I don't see much issue?

Reply:
Yeah it is ok to change. This is not stored anywhere so after a container restart it will use the new path without issues.

Reply:
This looks fine to me, and I'm assuming that somewhere we have plans to document the need for a restart of a container in 4.1?